raiden/ui/app.py

```diff
@@ -66,7 +66,8 @@ def _setup_matrix(config):
     if config["transport"]["matrix"].get("available_servers") is None:
         # fetch list of known servers from raiden-network/raiden-tranport repo
         available_servers_url = DEFAULT_MATRIX_KNOWN_SERVERS[config["environment_type"]]
-        available_servers = get_matrix_servers(available_servers_url)
+        available_servers = available_servers_url if \
```

Comment:
why we need to check the environment type here? 
if we use:

`         available_servers_url = DEFAULT_MATRIX_KNOWN_SERVERS[config["environment_type"]] `

the `DEFAULT_MATRIX_KNOWN_SERVERS` variable will point to the development or production ones already, right?

Reply:
It's because of the typing, if config["environment_type"] it's PRODUCTION then the format for available_servers_url it's the list of yml files to get the urls from, but for DEVELOPMENT we are using the server url directly so we don't need to parse it.

Reply:
We could improve this by removing the get_matrix_servers function call and modifying the format on PRODUCTION config value too and always using the final url values instead of yml files.

Reply:
Ok got it, its ok for me to leave it this way and iterate when the Production environment is used correctly.

Reply:
Sounds good to me